pkgs/development/tools/yarn-berry/default.nix

```diff
@@ -0,0 +1,35 @@
+{ fetchFromGitHub, lib, nodejs, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "yarn-berry";
+  version = "3.4.1";
+
+  src = fetchFromGitHub {
+    owner = "yarnpkg";
+    repo = "berry";
+    rev = "@yarnpkg/cli/${version}";
+    sha256 = "sha256-eBBB/F+mnGi93Qf23xgt306/ogoV76RXOM90O14u5Tw=";
+  };
+
+  buildInputs = [
+    nodejs
+  ];
+
+  dontConfigure = true;
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm 755 "$src/packages/yarnpkg-cli/bin/yarn.js" "$out/bin/yarn"
```

Comment:
Can we build this file manually?

Reply:
Okay, I'll try it.

Reply:
@dotlambda I've pushed a commit so that we build it manually.
Could you please take another look?